src/components/sbb-timetable-row/sbb-timetable-row.scss

```diff
@@ -7,6 +7,7 @@
 :host {
   --sbb-timetable-row-gap: var(--sbb-spacing-fixed-2x);
   --sbb-timetable-row-height: #{sbb.px-to-rem-build(25)};
+  --sbb-timetable-row-height-badge: var(--sbb-card-badge-height);
```

Comment:
does this work? card badge is only defined if timetable row would be use inside sbb-card-badge (?)